docs/INSTALL.md

```diff
@@ -0,0 +1,179 @@
+<div style="display: flex; align-items: center;">
```

Comment:
Some advanced CSS doesn't seems to work on GitHub, so instead of logo and heading on the same line, they are being presented on multiple lines.

Reply:
They are some more similar occurrences down below, kindly check them out!